### ISSUE TITLE
[21.4.3] Node tar ball missing

### ISSUE THREAD

**Issue by tuxmaster5000** [greenbone/gsa#3188]
For this release the node tar ball is missing at the download section of git hub.

**Comment by tgurr** [comment]
> Sorry for not remembering this issue!
> 
> I've uploaded static build files (gsa-dist-22.4.x.tar.gz) for https://github.com/greenbone/gsa/releases/tag/v22.4.1 and https://github.com/greenbone/gsa/releases/tag/v22.4.0. Hope that helps. Please let me know if something is wrong or still missing.

@bjoernricks Just want to say a special thank you for providing these `-dist` tarballs now, awesome! We've switched our package over to using the prebuilt static files for the latest release, that allowed us to get rid of node which was a really huge and time consuming buildtime dependency. Thanks and please never stop providing those.

**Comment by bjoernricks** [comment]
Yes I was also frustrated by building the huge pile of stuff and fiddling with yarn and node versions in our source build guide. And as I am a DevOps guy since more then year we heavily improved our CI/CD pipelines which allows us to do a lot more automation now and therefore the dists are build and uploaded during the release automatically.
